ecl/hql/hqlmanifest.cpp

```diff
@@ -52,8 +68,10 @@ class ResourceManifest : public CInterface
 
 public:
     Owned<IPropertyTree> manifest;
+    StringBuffer fileContents;
```

Comment:
Should this be a local variable rather than a member?

Reply:
No, because I output the contents verbatim to the archive.